src-docs/src/views/code/code_example.js

```diff
@@ -39,6 +41,23 @@ export const CodeExample = {
   title: 'Code',
   intro: (
     <>
+      <EuiCallOut title="EUI provides support for static code">
+        <p>
+          <strong>EuiCode</strong> and <strong>EuiCodeBlock</strong> are
+          intended to render static lines or blocks of code in read-only
```

Comment:
```suggestion
          intended to render static lines or blocks of code in <strong>read-only</strong>
```

Just to add emphasis